src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ CHANGELOG
 
 6.3
 ---
-
+ * Add support for reading objects properties with `Unique` constraint `fields` option
```

Comment:
You should not remove the newline between `---` and he first element of the list.
And usually, we add new element at the end of the list.

Reply:
According to https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry 
`New entries must be added on top of the list.`